pkgs/development/python-modules/libversion/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi, python, pkgconfig, libversion, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "libversion";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "18hhn7b7458lybs8z8ckh0idm7a2g4c4b5v2p9rr0lb618rchvds";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ libversion ];
+
+  doCheck = true;
```

Comment:
That's set automatically for python packages. I meant `libversion`, not `pythonPackages.libversion`

Reply:
Oh, okay! For `libversion` says: `make: *** No rule to make target 'check'.  Stop.`

Reply:
`checkTarget = "test"`